src/chi/components/drawer/drawer.scss

```diff
@@ -174,9 +174,11 @@ $animation-duration: 0.5s;
   }
 
   .-close {
-    position: absolute;
+    align-self: flex-end;
+    position: sticky;
```

Comment:
Not working. Can you use position: fixed? Sticky doesn't have enough browser support.
May need to set overflow: auto on drawer content to prevent the button from scrolling.